hypervisor/qemu/qemu_process.go

```diff
@@ -13,6 +14,80 @@ import (
 	"github.com/hyperhq/runv/hypervisor"
 )
 
+type QemuLogFile struct {
+	name   string
+	offset int64
+	eof    bool
+}
+
+func (f *QemuLogFile) Read(p []byte) (n int, err error) {
+	reader, err := os.Open(f.name)
+	if err != nil {
+		return 0, err
+	}
+	reader.Seek(f.offset, os.SEEK_SET)
+
+	for {
+		n, err = reader.Read(p)
+		f.offset += int64(n)
+
+		if err == io.EOF {
+			if f.eof {
+				reader.Close()
+				return
+			}
+
+			time.Sleep(1 * time.Second)
+			reader.Close()
+			reader, err = os.Open(f.name)
+			if err != nil {
+				reader.Close()
+				return
+			}
+			reader.Seek(f.offset, os.SEEK_SET)
+		}
+		if err != nil || n != 0 {
+			reader.Close()
+			return
+		}
+	}
+}
+
+func (f *QemuLogFile) Close() error {
+	f.eof = true
+	return nil
+}
+
+func (f *QemuLogFile) Watch(skipAll bool) {
+	if skipAll {
+		reader, err := os.Open(f.name)
+		if err != nil {
+			return
+		}
+		f.offset, err = reader.Seek(0, os.SEEK_END)
+		if err != nil {
+			return
+		}
+		reader.Close()
+	}
```

Comment:
the value of offset is saved and restored when associateQemu(), `skipAll bool` is unneeded, please remove it.